contracts/land/LANDRegistry.sol

```diff
@@ -479,4 +464,78 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     assembly { size := extcodesize(addr) }
     return size > 0;
   }
+
+  //
+  // LAND Ping
+  //
+
+  /**
+   * @dev Set the date from when the LAND Ping feature should be enabled
+   * @param _gracePeriod Desired amount of time in seconds from now to enable the feature
+   */
+  function setGracePeriod(uint256 _gracePeriod) external onlyDeployer {
+    require(_gracePeriod != 0, "Grace period can not be 0");
+    // solium-disable-next-line security/no-block-members
+    gracePeriod = block.timestamp.add(_gracePeriod);
+    emit GracePeriod(msg.sender, gracePeriod);
+  }
+
+  /**
+   * @dev Set the amount of time that should pass for a LAND to be transferred to
+   * a new onwer
+   * @param _deemPeriod Desired amount of time in seconds for a LAND to decay
+   */
+  function setDeemPeriod(uint256 _deemPeriod) external onlyDeployer {
+    require(_deemPeriod != 0, "Deem period can not be 0");
+    deemPeriod = _deemPeriod;
+    emit DeemPeriod(msg.sender, deemPeriod);
+  }
+
+  /**
+   * @dev Ping an address
+   * @param _user address of LAND holder to be pinged
+   */
+  function ping(address _user) external {
+    require(
+      _isApprovedForAll(_user, msg.sender) ||
```

Comment:
As we talked offline. Let's add the ability for the `proxyOwner` to update it too